docs/blog/2023-09-06-distributed-key-generation.md

```diff
@@ -0,0 +1,154 @@
+
+**🏴‍☠️ First Step:** Each of the pirates (Pirates A, B, C, and Jack) agrees to jointly create a secret key to a treasure box. But none of them should have the complete key on their own.
+
+**🏴‍☠️ Generation:** Jack creates a polynomial of a degree t and distributes shares to the other pirates including himself. A does the same, and B and C follow too.
+
+That means that at the end, each pirate will have four shares created including his/her own.
+
+🙍🏾Jack : {s_Jack, s_A, s_B, s_C}    🙍🏾‍♂️A: { s_A, s_Jack, s_B, s_C}
+
+🙍🏾‍♀️B: { s_B, s_Jack, s_A, s_C}         🥷🏾C: {s_C, s_Jack, s_A, s_B}
+
+**🏴‍☠️ Sharing:** As soon as the above is complete, each pirate distributes their shares to the other pirates. So, at the end of this, each should have a share from the others. 
+
+![graph](./images/2023-09-06-dkg/graph.jpg)
+
+**🏴‍☠️** The next step requires that each pirate aggregates their shares. This one will be their final share that they need for the key. 
+
+**🏴‍☠️ Reconstruction:** Based on a given threshold, the pirates can reconstruct the secret. What this eventually means is that a subset of pirates more or equal to a specific threshold t can combine their final aggregated shares to reconstruct the polynomial and thus the secret.
+
+![hooray.gif](./images/2023-09-06-dkg/hooray.gif)
+
+
+## Decentralization:
+
+The beauty of DKG is its decentralized nature. Since every pirate contributes equally to the randomness of the key, it's incredibly difficult for an outside adversary to determine the full secret key without access to the requisite number of shares. Moreover, the process guarantees that even if a pirate tries to cheat and distribute false shares they can be detected and potentially excluded from the protocol.
+
+Overall, DKG algorithms are an amazing research area and they can be used securely in a variety of cryptographic operations, like generating secret keys for signing or validating. Furthermore, by allowing DKG to be placed,  the power is spread among the participants, reducing the risk of a single point of failure and increasing the overall security of the network.
+
+# Drand’s DKG Approach:
+
+[Drand](https://drand.love/) incorporates [Pedersen's DKG scheme](https://www.cs.cornell.edu/courses/cs754/2001fa/129.PDF), an advanced framework originally devised by Torben Pryds Pedersen. This method ingeniously builds upon multiple instances of [Feldman's Verifiable Secret Sharing (VSS)](https://www.cs.umd.edu/~gasarch/TOPICS/secretsharing/feldmanVSS.pdf), enhancing them with added verification procedures to further secure the protocol.
+
+From a high-level perspective, the protocol can be divided into three distinct phases:
+
+1. **Share Distribution:**  Each participant generates a random secret and distributes it to the other participants using Feldman's VSS. Then, each party sends a unique share to every other member and publicly broadcasts a list of commitments.
```

Comment:
```suggestion
1. **Share Distribution:** Each participant generates a random secret and distributes it to the other participants using Feldman's VSS. Then, each party sends a unique share to every other member and publicly broadcasts a list of commitments.
```

Reply:
Could be worth clarifying what a 'commitment' is, in case people are noobs

Reply:
I am not sure how  to explain commitment :p Like a commit value? Any ideas welcome

Reply:
I'd say with the envelope analogy, maybe?